### ISSUE TITLE
Automation UI behavior breaking change on GetClickablePoint for menu items when compiling code with .Net 4.8

### ISSUE THREAD

**Issue by dfev77** [dotnet/winforms#4943]
Not sure if posting to right repo but as https://github.com/microsoft/dotnet/ seems to be retired, I'm posting it here. Please let me know if this should be open on another repository

* .NET Core Version: none

* Have you experienced this same bug with .NET Framework?: .Net 4.8

**Problem description:**
When WinForms application is compiled using .Net 4.8 (as opposed to 4.7.2), Automation is not able to retrieve the clickable point through IUIAutomationElement.GetClickablePoint or IUIAutomationElement.GetCurrentPropertyValueEx(UIA.UIA_PropertyIds.UIA_ClickablePointPropertyId)
You need to disable some of the accessibility changes in 4.8 to have this working
`  <runtime>
  <AppContextSwitchOverrides value="Switch.UseLegacyAccessibilityFeatures=true;Switch.UseLegacyAccessibilityFeatures.2=true;Switch.UseLegacyAccessibilityFeatures.3=true" />
  </runtime>`

**Expected behavior:**
Automation should work without having to disable accessibility features.

**Minimal repro:**
See attached solution. The code includes projects cloned from https://github.com/FlaUI/FlaUI, it was added here just for ease of the reproduction steps.
For where the issue occurs see TryGetClickablePoint method in FlaUI.UIA3\UIA3FrameworkAutomationElement.cs line 147
To use the solution, there is a test WinForms application and a console app to do the automation request.

[Net48MenuItemAutomationIssue.zip](https://github.com/dotnet/winforms/files/6510532/Net48MenuItemAutomationIssue.zip)

**Comment by merriemcgaw** [comment]
Thank you for bringing this to our attention. Unfortunately, the .NET Framework 4.8 is considered "done" and is only taking security fixes or those fixes coming through our Customer Service programs. So the likelihood of this being fixed in .NET Framework 4.8 is small, but a fix in .NET 6 is quite likely if it repros in the current version. We are actively taking fixes and contributions from the public when it comes to .NET 6. Note that WinForms was not originally created to support UIA and a lot of limitations exist in that space. During .NET Framework 4.7-4.8 releases we added a lot of support for various UIA patterns, but the support was far from complete. We are working on addressing those limitations in .NET Core 3.0 and onward.

Can you let us know if you have tested this out on .NET 6? If so please let us know and we'll get a fix for this in our next release.

**Comment by dfev77** [comment]
@merriemcgaw thanks for the reply. Our up is currently targeting 4.7.2 and we are making use of UIAutomation in our tests through FlaUI lib. It works fine, the issue was observed through our migration to 4.8, too bad we did not do is sooner :)

.Net 6 seems to work fine in regards to this.
(just that the window is 2x in size, same on .net 5, but that is a different topic)

**Comment by merriemcgaw** [comment]
I think we're going to close this issue given that it applies to .NET Framework rather than the .NET 6 product. 
@dfev77 When it comes to the larger form size, my suspicion is that because our templates include setting the DPIAwareness to System Aware.
